### PR TITLE
v8.3.0: update learning path

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/br-snapshot-guide.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ tiup br validate decode --field="end-version" \
 > - 从 BR v7.6.0 开始，为了解决大规模 Region 场景下可能出现的恢复瓶颈问题，BR 支持通过粗粒度打散 Region 的算法加速恢复（实验特性）。可以通过指定命令行参数 `--granularity="coarse-grained"` 来启用此功能。
 > - 从 BR v8.0.0 开始，通过粗粒度打散 Region 算法进行快照恢复的功能正式 GA，并默认启用。通过采用粗粒度打散 Region 算法、批量创建库表、降低 SST 文件下载和 Ingest 操作之间的相互影响、加速表统计信息恢复等改进措施，快照恢复的速度有大幅提升。在实际案例中，快照恢复的 SST 文件下载速度最高提升约 10 倍，单个 TiKV 节点的数据恢复速度稳定在 1.2 GiB/s，端到端的恢复速度大约提升 1.5 到 3 倍，并且能够在 1 小时内完成对 100 TiB 数据的恢复。
 > - 从 BR v8.2.0 开始，命令行参数 `--granularity` 被废弃，粗粒度打散 Region 算法默认启用。
+> - 从 BR v8.3.0 开始，快照恢复增加了对 TiKV 和 TiFlash 可用磁盘空间的检查，包括在开始阶段会根据需要恢复的 SST 文件大小检查 TiKV 和 TiFlash 是否有足够的磁盘空间，以及在 TiKV（v8.3.0 或之后的版本）下载每个 SST 文件前会检查 TiKV 是否有足够的磁盘空间。如果任一阶段检测到可用空间不足，恢复任务会报错失败。你可以通过设置 `--check-requirements=false` 来选择跳过恢复任务开始阶段的检查，但 TiKV 在下载 SST 文件前的磁盘空间检查无法跳过。
```
Comment (hfxsd) [inline br/br-snapshot-guide.md:71]:
```suggestion
```

File: pd-control.md [hunk 2]
```diff
@@ -1322,6 +1322,12 @@ scheduler config balance-hot-region-scheduler  // 显示 balance-hot-region 调
 
     当一个 `evict-leader-scheduler` 的所有 store 配置都被移除后，该调度器也会自动被移除。
 
+- 在已有 `evict-leader-scheduler` 时，使用 `set batch` 子命令修改 `batch` 值。其中，`batch` 用于调整单次调度过程中生成的 Operator 数量，默认值为 `3`，取值范围为 `[1, 10]`。`batch` 值越大，调度速度越快。
+
+    ```bash
+    scheduler config evict-leader-scheduler set batch 10 // 设置 batch 值为 10
+    ```
+
```
Comment (hfxsd) [inline pd-control.md:1330]:
```suggestion
```

File: _index.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: TiDB 产品文档
-aliases: ['/docs-cn/dev/','/docs-cn/','/docs-cn/stable/']
+aliases: ['/docs-cn/v8.3/','/docs-cn/','/docs-cn/stable/']
```
Comment (qiancai) [inline _index.md:3]: 这行看起来不需要在这个 PR 里改，因为在 https://github.com/pingcap/docs-cn/pull/18392/files 里会删除它

File: _index.md [hunk 4]
```diff
@@ -10,129 +10,129 @@ summary: TiDB 是 PingCAP 公司自主设计、研发的开源分布式关系型
...
 
-[PingCAP Clinic](https://docs.pingcap.com/zh/tidb/dev/clinic-introduction)
+[PingCAP Clinic](https://docs.pingcap.com/zh/tidb/v8.3/clinic-introduction)
 
 </LearningPath>
 
 <LearningPath label="参考" icon="cloud-dev">
 
-[TiDB 路线图](https://docs.pingcap.com/zh/tidb/dev/tidb-roadmap)
+[TiDB 路线图](https://docs.pingcap.com/zh/tidb/v8.3/tidb-roadmap)
```
Comment (qiancai) [inline _index.md:123]:
```suggestion
[TiDB 路线图](https://docs.pingcap.com/zh/tidb/dev/tidb-roadmap)
```

File: _index.md (hunk 3 above)
Comment (qiancai) [inline _index.md:3]:
```suggestion
aliases: ['/docs-cn/dev/','/docs-cn/','/docs-cn/stable/']
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**